wolftpm/doxygen.h

```diff
@@ -0,0 +1,63 @@
+/** @defgroup wolfTPM2_Wrappers wolfTPM2 Wrappers
+ *
+ * This module describes the rich API of wolfTPM called wrappers.
+ *
+ * wolfTPM wrappers are used in two main cases:
+ * * Perform common TPM 2.0 tasks, like key generation and storage
+ * * Perform complex TPM 2.0 tasks, like attestation and parameter encryption
+ *
+ * wolfTPM enables quick and rapid use of TPM 2.0 thanks to its many wrapper functions.
+ *
+ */
+
+/** @defgroup TPM2_Proprietary TPM2 Proprietary
+ *
+ * This module describes TPM2 commands specific only to wolfTPM.
+ *
+ * Typically, these commands include helpers for handling TPM 2.0 data structures.
+ *
+ * There are also functions to help debugging and testing during development.
+ *
+ */
+
+/** @defgroup TPM2_Standard TPM2 Standard Commands
```

Comment:
I don't see this group of `TPM2_IO` used or have any API. Is this on purpose?

Reply:
Dimi is working on group `TPM2_IO` and will be in a followup PR.